libexec/python/utils.py

```diff
@@ -172,19 +172,22 @@ def api_get(url,data=None,default_header=True,headers=None,stream=None,return_re
 
     return stream
 
+
 def basic_auth_header(username, password):
     '''basic_auth_header will return a base64 encoded header object to
     generate a token
     :param username: the username
     :param password: the password
     '''
     s = "%s:%s" % (username, password)
+    logger.debug("AUTH: Input username/password: %s", s)
```

Comment:
We don't need to put debug statements everywhere!
This code is simple enough, if we fix the bug we currently have, I doubt we'll see any other bug related to this code in the future. So the debug code won't be needed. Well, hopefully.